src/sfma/api.py

```diff
@@ -134,8 +145,56 @@ def __init__(self,
         self.inefficiencies = np.zeros(self.data.num_obs)
 
     def fit(self, **kwargs):
-        self.solver.fit(x_init=self.x_init, data=self.data, **kwargs)
+        options = kwargs.get('options', {'solver_options': {}})
+        self.trimming.fit(x_init=self.x_init, data=self.data, n=len(self.data.obs),
```

Comment:
Will be nice if we could overwrite `self.x_init`, really like what you did for `options` we could follow that do `x_init = kwargs.get("x_init", self.x_init)`

Reply:
1b6d642 -- switched to `kwargs.pop` :)